awxkit/awxkit/api/pages/credentials.py

```diff
@@ -1,4 +1,5 @@
 import logging
+from urllib.parse import urljoin 
```

Comment:
We need this module to be compatible with python2.7 for the time being. In python2.7, `urljoin` lives in `urlparse`. 

Since this is a built-in library function in either version of python, one way to make this compatible for python2 and python3 would be to wrap this in a try-except

```python
try:
    from urllib.parse import urljoin
except ImportError:
    from urlparse import urljoin
```

Reply:
Thanks for your review @jakemcdermott,  I will update properly.